tests/intern.ts

```diff
@@ -1,9 +1,17 @@
 import intern = require('intern');
+import has = require('intern/dojo/has');
+import topic = require('intern/dojo/topic');
+import echo = require('intern/dojo/has!host-node?./services/echo');
 
+if (has('host-node') && intern.mode === 'runner') {
+	echo.start().then(function (server: any) {
+		topic.subscribe('runner/end', server.close);
```

Comment:
The topic needs an initial forward slash ("/runner/end"). Also, `server.close` should be called in a callback.